### ISSUE TITLE
`helm init -c` will add stable repo sliently

### ISSUE THREAD

**Issue by gyliu513** [kubernetes/helm#2810]
```
# helm init --client-only
Creating /home/sameersbn/.helm 
Creating /home/sameersbn/.helm/repository 
Creating /home/sameersbn/.helm/repository/cache 
Creating /home/sameersbn/.helm/repository/local 
Creating /home/sameersbn/.helm/plugins 
Creating /home/sameersbn/.helm/starters 
Creating /home/sameersbn/.helm/cache/archive 
Creating /home/sameersbn/.helm/repository/repositories.yaml
# helm repo list
NAME  	URL                                             
stable	https://kubernetes-charts.storage.googleapis.com
local 	http://127.0.0.1:8879/charts
```

We should add some output telling customer that the `helm init` will also add two default repos.

**Comment by sharno** [comment]
Hope to get the [pull request](https://github.com/kubernetes/helm/pull/2928) reviewed

**Comment by bacongobbler** [comment]
fixed via #2928.
